### ISSUE TITLE
Is this Menu going to be updated for the new update?

### ISSUE THREAD

**Issue by RhinoGamin** [infamouscheats/InfamousCheats#17]
I would like to know when and if this menu is going to be updated. i have the platinum lifetime membership. is this menu ever going to be active again?

**Comment by Exophobias** [comment]
No one knows. The inf dev team has been quiet about everything.

**Comment by ImGrayy** [comment]
Could be never. They were hit with a copyright claim from T2. I suppose they were forced to shut it down, nothing yet on if we are going to get it back. If we do they have a lot of work to catch up on anyways.

**Comment by ghxost** [comment]
These other menus suck man :P plus I'm getting reports showing up now... I never realized just how special infamous was in that regard, never one single report, EVER and I am obvious just not a dick...

PLEASE release a purely defensive menu at least haha!

**Comment by ImGrayy** [comment]
That's true, Infamous IMO was the best menu ever. Absolute comes second. If you want a menu that comes close to Inf try Absolute. Next update blocks reports.

**Comment by ghxost** [comment]
I can vouch for all but next update promises lol. AbS Good stuff though

**Comment by holyr00d** [comment]
how close to inf is abs? i was checking out engh and was planning on getting premium there.

also thinking of just creating own custom menu. all the tools are out there... grabbed most of them.

**Comment by ghxost** [comment]
Do it up if you can. ABS is as polished but lacks some of the great features like report redirection - rumored to be forthcoming...
The ASI loader works great too, have another menu was able to rename to ASI and just load with ABS by placing other one in directory

**Comment by ImGrayy** [comment]
@ghxost Not rumored, it is true and confirmed by a staff member of Abs. Also yes, VIP is absolutely worth it.

The menu does lack a lot of features, but hopefully they will come around at some point or maybe even in the upcoming update.

**Comment by ghxost** [comment]
So much for that......

No more updates

Damn

**Comment by infamouscheats** [comment]
As people can see, things are still pinched during these times.  The game company is still very proactive with their search and destroy methods.  Take this for what it is.  Here's an excerpt from an ending post from another website.

"This is your last chance to keep the **** until *.**, once the authorization server goes offline by the end of today (August 13th, 11 PM PST) you will be unable to use the **** unless you've logged onto *** at least once with version *.*.*.* . You can check your version by looking at ********-****.***. Keep in mind that when it goes down it'll be locked to the last PC you've logged on with, and if you reformat that PC, you'll lose it."

**Comment by ghxost** [comment]
2take1 is sti kicking

Fyi

Small but stable

Glad all, hope to see inf. Back under another name...

**Comment by ghxost** [comment]
Unreal and Evolve are getting a lot of recommends on abs forum for alternatives
No idea, just FYI to those still around here.....

**Comment by Exophobias** [comment]
I tried adding you on Steam and you blocked me! You made me feel abandoned xD. I don’t suppose you can give hints or winks if anything were to happen later on with inf or whatever  new name comes about.

________________________________

**Comment by ImGrayy** [comment]
@ghxost I've owned Evolve, I didn't really like it that much due to the staff and the huge delay on updates. It takes them nearly 2+ weeks to get the menu up and running. A lot of features you think that would be base features are not and are in the VIP section. They do not accept Paypal so you have to be charged an additional fee, adding all up to a nearly $60 menu. It doesn't seem worth it at all. I've gotten banned by a suggestion. lol

**Comment by ImGrayy** [comment]
@ghxost Discord?

**Comment by Exophobias** [comment]
Huh?

________________________________

**Comment by infamouscheats** [comment]
As a party who can say "some" stuff about what happened in this situation, there has been a somewhat settlement.  There may or may not be something in the future.  This is not the platform to say.  You can read a snippet of what's been happening in this article from TF.  https://torrentfreak.com/gta-vs-take-two-wants-lying-cheat-maker-to-pay-up-180901/

**Comment by Exophobias** [comment]
Will you be able to give us a hint of the new platform it will be on? What about that one site that the launcher connects to (Wiresharked it)
Also, is it true that there is a working version out there? A few weeks back I heard rumors of some people posting links of a download for it.

________________________________

**Comment by holyr00d** [comment]
thanks for the update man.

**Comment by infamouscheats** [comment]
@Exophobias Information on platforms like this can be easily monitored by anyone who cares to find it.  Unfortunately, there were, and always will be, real consequences to actions.  It burns me to say this but don't rely on anything that once was.  I can say that the latest version *was* operational to a point, and it was the latest version linked on a few different sites.  I can't say if/how many still have access and I can't comment on future platforms and websites used to connect to users.  I do appreciate all interest, concerns, questions, and comments. Feel free to talk around  yourselves here. If anything were to ever change in the future, this site would probably be the avenue to find info. ^B

**Comment by holyr00d** [comment]
if you have your own web server you should really consider setting up shop on tor with onion address. or at least discuss it. you will have to no longer accept emails from hotmail, outlook etc ATP (advanced threat protection) is in full effect. cannot open links in emails from sites related to our subject and even mails from github with links https://nam01.safelinks.protection.outlook.com blocking them all now.

which is quite annoying as my account was regd with outlook mail account...

**Comment by infamouscheats** [comment]
@R00dNet Having our own server wasn't a big deal.  Using the server to auth simply in itself wasn't/isn't a problem.  Some other things are in play even with precautions, which is why now isn't the best time to consider any other avenue.  About Outlook, a lot of people agree with you that MS protection is going in an inconvenient direction.

**Comment by M588** [comment]
i know that a selective group are still able to gain access to it and are using it i am very disappointed that having purchased the vip i can no longer use it and they can so when pepole talk about how bad it is i dont belive a word of it if it really is that bad why why why are they still useing it and allso in public

**Comment by jWXZSCsqrpZRSVwauBjn** [comment]
You accepted the TOS before buying in which they clearly said that they can shut down their service at any time. If you're in court against a multi million dollar company you wouldn't do anything else. Even if a few people still have access to it you don't have any reason to cry.

**Comment by M588** [comment]
you really dont know do you read my post carefully and you see the light the pepole who are still using it

**Comment by holyr00d** [comment]
@m588 proof? otherwise you are just talking nonsense. I have not seen or experienced users using infamous. some menus still going have implemented some anti-grief measures same as what infamous had. no one has reported infamous is still working and i dont believe all would remain silent if it were as the haves like to boast to the have nots.

**Comment by ghxost** [comment]
Lol, proof...

**Comment by M588** [comment]
after infamous went down there was a leaked version Infamous_2.5.4.11 but now called  paradice still with infamous files so yea i do know then another leak 2 weeks later InfamousV_Premium_2.5.6.536 then i week later the addmin and staff who are still using it blocked all vip users that got the leak i wont post the users name but guess what mom is strill using it and i mean mom one of the devs from imfamous so if she is still using it and in a public session and there are others why have we been left out in the cold and why is it so bad that we who paid can not get the update im done talkingf i dont wont to say anymore as i might reveal more info if the imfamous paradice is done then why are they updating it and using it in a public session there are invite only sessions they could use those

**Comment by ghxost** [comment]
You dont mod in private sessions...

Well I don't think its wise anyways

And if it's just them using the new version who cares?

You write your own and you can be the only one using it.

Maybe its testing. Maybe the block was for a reason you dont comprehend.

Give it a rest already geez

**Comment by M588** [comment]
![untitled](https://user-images.githubusercontent.com/43043509/45177458-f6830980-b20a-11e8-868f-f5fbe2c996cf.gif)

**Comment by ghxost** [comment]
I had to reformat so I'm boned anyway haha

**Comment by M588** [comment]
if you had used hwid identifer first and then you could have used hwid changer you could have got it to work again well that is if they realised the update

**Comment by ghxost** [comment]
Meh

If it comes back hopefully it will be untied again

I didnt know what hwid was used

You can tie it to any piece of hardware or unique I'd generated by install.
Not like there was a guide saying hey, we locked it to your hard drive id

If not no biggie

Others exist and I have them. This was just my preferred one, obviously

**Comment by Exophobias** [comment]
New leaked files are not HWID bound because their authentication server was up and so the auth.dat file wasn't needed and could get reauthenticated.

In other words anyone with a valid inf account could use it.
________________________________

**Comment by ghxost** [comment]
I only got the first leak :(

**Comment by holyr00d** [comment]
@M588 my apologies. interesting info to know.. thank you for verifiying. this is indeed somewhat disappointing. although it does not necessary mean vips have been forgotten or ripped off. It could well be in testing pending release....  a dev using their menu, literally, in game doesnt mean a lot. they designed it. if it were me id still be using it too.

**Comment by ImGrayy** [comment]
This is extremely unfair to both Infamous and Benny. You must realize what is currently going on with them firstly, instead of attacking him to release it to the public. Everyone here should just accept this and wait it out. Like Benny said, he can't answer it now but he did say there wasn't a definite yes or no to this yet.

**Comment by M588** [comment]
@RhinoGamin no one is attacking anyone i am just stating that if the infamous has stopped all updates due to the court case and the penallty to keep updating and realising the updates to the vips as well as the primium members is so BAD so then why has the name changed to paradice and why is there 3 updates ie 2.5.6.536 , 2.5.4.11 and the new update witch has not been leaked so noone at the moment  knows  the version number is still being used by the selective few in a public session for all to see surley if they get detected then its all back to court ? so its not really that BAD  otherwise they would not use it and please dont say its undetectable nothings undetectable useing mods online allways comes with a risk of being detected

**Comment by Exophobias** [comment]
Because there IS three updates, 2.5.6.78 and the latest was 2.5.6.536

Paradise – When the menu opens. The name shows as Paradise, but if you look at all the files (You could probably easily find the new 2.5.6.78 leak) – The files say Infamous.

Those were working versions on GTA 1.44 and any infamous account (Who has paid. Premium or Platinum) could be used for it. That has now changed and now only authorized accounts can. Anyone who is not authorized gets their GTA crashed for NULLPTR

________________________________

**Comment by holyr00d** [comment]
can someone please advise the direction to be headed for new sub then? irc? telegram?

**Comment by M588** [comment]
ok im done ill give 7 days notice from today if there is no official statement from the devs Infamous ? paradice  i will supply R with  proof of futher development  and leaks 

TIME TO SPEAK  MUM OR OR ILL LEAK IT ALL

**Comment by ImGrayy** [comment]
@M588 Guess what Ryan told me was right. This is in all seriousness. Please just calm down, we're not going to get anywhere if you "tell R*". That threat only makes things worse for you and for EVERYONE involved. R* is more than likely already watching this repository because everyone just blows a lot of information out of their ass that should just be kept to yourself if told privately. Just be glad Benny is still talking to us about what's going on, even under his current condition.

Benny has said previously, there may or may not be something. So don't jump to conclusions when there  hasn't even provided an answer yet please. Just consider on why he's not giving a full answer, if he does it'll fuck him over and fuck all of us over.

**Comment by jWXZSCsqrpZRSVwauBjn** [comment]
@M588 Here you'll find everything you want to know. https://www.pacermonitor.com/public/case/24131026/TakeTwo_Interactive_Software,_Inc_v_Cameron_et_al

**Comment by holyr00d** [comment]
@m588 thretening devs really? you sound like a spoilt brat. grow up. what you have to be proven is nothing really. you cannot prove anything. you have a screenshot and some files good for you. R* has already won in regards with infamous as infamous is no longer available and on clearnet so therefore no clearnet home =  no website = no sales also = no-one to send cease and desist to. R* does not care about the devs still using inf menu. they got all the main users that is a big win for R*. your threat means absolutely nothing. really. from a legal stand point as of right now, absolutely nothing.

**Comment by ghost** [comment]
@M588 maybe you need to experience what the devs are going through in reality to fully understand what's going on behind the scenes since reading black and white (from all those court links others are posting) seems to get you nowhere. 
 
they knew what they were getting into when they began coding and selling inf to the public. as from the customer standpoint, you are expected to know what you're getting into as well when purchasing something such as this - which is also why you agreed to the ToS before the purchase or any such purchases ever. 
 
i personally purchased plat lifetime as well (ryan has my proof even though i left the discord as i don't really use it much) and i'm not even salty as to why i didn't know of Paradise and/or whatever 3 updates you speak of.. basically just kept quiet and (still) patiently waiting for (hopefully) a future with inf sometime again. you should count yourself lucky that you even got to hear of Paradise, those 3 updates and even use them. that being said, since you lost your access it might mean something happened and they needed to narrow down their trusted users or some shit. 
 
threatening openly like that doesn't do harm to anyone but yourself really.. it just makes me (if from a dev/seller standpoint) to not give you access ever to any other menu even if you paid me because you're simply one of those potential hazards that might bring the whole discreet operation down. learn some perspective, it will do you good in life.

**Comment by Exophobias** [comment]
https://www.cnbc.com/2018/10/18/gta-v-cheat-developers-homes-raided-by-rockstar-take-two-interactive.html

Welp, I don't think Infamous is EVER coming back if that site is correct.

**Comment by infamouscheats** [comment]
It's unfortunate and true that such an event and spectacle has been carried out by the parent company.  We are near the end of the life cycle of the game, so we should just face the hard fact that an adjacent project may not come into fruition.  I can't comment on true details of what's happening but we are still a family and as you know, families are strong.

**Comment by Exophobias** [comment]
That's a shame, Take-Two are cunts. I understand it's their right and their property but its overkill. They talk about lost revenue but no one was going to buy those shark cards anyways because they are a scam.
Ever since inf went down I have not played GTA, and I won't play GTA without a good menu and well there is none other than Infamous itself. I'll be honest, I was lucky and got the "private" access to the new files and had them as of late August until it fully went down and now just crashes GTA.

I don't plan to play GTA again, I lived without it for years because my Steam account GTA is perm banned.

Thanks for one hell of a ride Benny and inf team! I hope Take-Two doesn't fuck you or any of the team's lives over too much (I can't expect no interference with anyone's lives with the current court orders)

**Comment by holyr00d** [comment]
that is unreal. the precedent this is going to set is something else. Wonder how much it cost to get order to raid civilians homes, something you usually only associate with law enforcement,.

**Comment by Ilon3** [comment]
> lopers.pdf.lzo.xz.7z.gz.tar
Unknown
I need this opened.

**Comment by Exophobias** [comment]
??

If you tried to send a file on the comment, it doesn’t work.

________________________________

**Comment by sghosh1810** [comment]
> > lopers.pdf.lzo.xz.7z.gz.tar
> > Unknown
> > I need this opened.

Upload to Mega/zippyshare and send the link

**Comment by ImGrayy** [comment]
I'm just going to assume you wanted to see what was inside of it. It's just a bunch of papers that involve basically everyone Infamous and Take-Two requesting ALL infamous related files from GITHUB from various users on the site here. So basically: Take-Two is still fucking everyone from infamous in the ass and operating on GITHUB is no longer going to happen.

EDIT: It's probably for the best that this is no longer discussed here. I'm not sure what could possibly happen with this information being available to the public. This page will likely be deleted at some point in the future by GITHUB themselves.

**Comment by infamouscheats** [comment]
Yes.  We are still limited in discussion.  But, we are still here listening (reading) things from users.

**Comment by Exophobias** [comment]
> Yes. We are still limited in discussion. But, we are still here listening (reading) things from users.

Honestly, I'm surprised with everything happening, houses raided, fines and whatnot; that you are still reading GitHub. Surprising.

**Comment by Exophobias** [comment]
> I'm just going to assume you wanted to see what was inside of it. It's just a bunch of papers that involve basically everyone Infamous and Take-Two requesting ALL infamous related files from GITHUB from various users on the site here. So basically: Take-Two is still fucking everyone from infamous in the ass and operating on GITHUB is no longer going to happen.
> 
> EDIT: It's probably for the best that this is no longer discussed here. I'm not sure what could possibly happen with this information being available to the public. This page will likely be deleted at some point in the future by GITHUB themselves.

Wait, Take-Two is requesting files from users on GitHub? Interesting.

**Comment by ImGrayy** [comment]
> Wait, Take-Two is requesting files from users on GitHub? Interesting.

According to the things I've read from what that person wanted to see, yes. I can provide more information or even proof on that if needed, but regarding Benny's response I don't think more information or proof is needed.

**Comment by Exophobias** [comment]
I’d personally like to see proof because of my curiosity on that subject. Rockstar digging in. I hate them


________________________________

**Comment by ImGrayy** [comment]
email? discord works too, might be in your dms still you can find me

**Comment by Exophobias** [comment]
Still the same or has it changed? Your name doesn’t show up when I search it.



________________________________

**Comment by ImGrayy** [comment]
#0156

**Comment by infamouscheats** [comment]
Hey, everyone.  I've been told that people involved with other popular tools are being sought out and legal action is still being taken heavily against those individuals.  The game company is still churning heavy profits from their in-game currency, which is why they are still going great lengths to shut down those other tools.

The other individuals who where apart of and/or created the other tools are getting leaned on pretty heavily, sadly.

**Comment by Exophobias** [comment]
Is anything new planned? Or is it for sure dead?

**Comment by holyr00d** [comment]
some time has passed now. is it possible to get directed to new home?

**Comment by ImGrayy** [comment]
Check out https://infamouscheats.cc again. This menu is not going to be updated. Look for other menus if you want to use one still. It's just going to be a chain reaction of developers taking source code and remaking menus and getting shut down.

**Comment by holyr00d** [comment]
this menu is never going to be updated. I am talking about inf new home new rebrand. ive found another paid, id rather be with inf devs though. there will be no chain reaction. nothing has changed. internals, externals paid and free all still in abundance a rife in-game. if anything take two have given up on gta online and are focusing on the next gta/gta online version.

**Comment by v0idp** [comment]
void#3903
